### PR TITLE
Upgrade packaging

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,20 +8,14 @@ language: python
 # NOTE: Explicit Python versions make Travis job description more informative
 matrix:
   include:
-    - env: TOX_ENV=py27
-      python: "2.7"
-    - env: TOX_ENV=py34
-      python: "3.4"
-    - env: TOX_ENV=py35
```
Comment (dblenkus) [inline .travis.yml]: This commit also removes support for Python 3.5, not only Python 2

File: setup.py [hunk 2]
```diff
@@ -77,6 +77,7 @@
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
```
Comment (dblenkus) [inline setup.py]: This should go into a separate commit - together with changes in tox file.

File: .travis.yml (hunk 1 above)
Comment (mstajdohar) [reply .travis.yml]: I have changed to commit message
